bundle/tests/presets_test.go

```diff
@@ -0,0 +1,27 @@
+package config_tests
+
+import (
+	"testing"
+
+	"github.com/databricks/cli/bundle/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPresetsDev(t *testing.T) {
+	b := loadTarget(t, "./presets", "dev")
+
+	assert.Equal(t, "myprefix", b.Config.Presets.NamePrefix)
+	assert.Equal(t, config.Paused, b.Config.Presets.TriggerPauseStatus)
+	assert.Equal(t, 10, b.Config.Presets.JobsMaxConcurrentRuns)
+	assert.Equal(t, true, *b.Config.Presets.PipelinesDevelopment)
+	assert.Equal(t, "true", b.Config.Presets.Tags["dev"])
+	assert.Equal(t, "finance", b.Config.Presets.Tags["team"])
+	assert.Equal(t, "false", b.Config.Presets.Tags["prod"])
+}
+
+func TestPresetsProd(t *testing.T) {
+	b := loadTarget(t, "./presets", "prod")
+
+	assert.Equal(t, false, *b.Config.Presets.PipelinesDevelopment)
+	assert.Equal(t, "true", b.Config.Presets.Tags["prod"])
```

Comment:
Please test that also the "team" tag survives. The test would pass if it didn't and something else added a target-named tag.

Reply:
I'm actually not sure I understand your comment. The test already says

```
assert.Equal(t, "finance", b.Config.Presets.Tags["team"])
```

Is that what you're looking for?

Reply:
Yes, this assertion should also happen in the prod test. It is only made in the dev test now.

Reply:
I see, added that to other test!